tests/pc/test_po_update.py

```diff
@@ -5,12 +5,14 @@
 
 import pytest
 import logging
-
+from tests.common.reboot import reboot
```

Comment:
nit: sort imports

Reply:
@arlakshm 
Will Incorporate this Comment